backend/bitboxbase/updater/updater.go

```diff
@@ -25,45 +28,67 @@ import (
 	"github.com/digitalbitbox/bitbox-wallet-app/util/logging"
 	"github.com/digitalbitbox/bitbox-wallet-app/util/observable"
 	"github.com/digitalbitbox/bitbox-wallet-app/util/observable/action"
+	"github.com/flynn/noise"
 
 	"github.com/gorilla/websocket"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	opICanHasHandShaek          = "h"
+	opICanHasPairinVerificashun = "v"
+)
+
+// BaseMiddlewareInfo holds some sample information from the BitBox Base
+type BaseMiddlewareInfo struct {
+	Blocks         int64   `json:"blocks"`
+	Difficulty     float64 `json:"difficulty"`
+	LightningAlias string  `json:"lightningAlias"`
+}
+
 // Updater implements observable blockchainInfo.
 type Updater struct {
 	observable.Implementation
-	blockchainInfo string
-	log            *logrus.Entry
-	running        bool
-	address        string
+	middlewareInfo      *BaseMiddlewareInfo
+	log                 *logrus.Entry
+	running             bool
+	address             string
+	bitboxBaseConfigDir string
+
+	baseNoiseStaticPubkey         []byte
+	channelHash                   string
+	channelHashAppVerified        bool
+	channelHashBitBoxBaseVerified bool
```

Comment:
sometimes it is named bitBoxBase, sometimes just base (a few lines above). would be nice to have the same naming everywhere.